docs/blog/2018-09-24-reach-router/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: How we improved Gatsby's accessibility in v2 by switching to @reach/router
+date: 2018-09-24
+author: Amberley Romo
+tags: ["routing", "v2", "accessibility"]
+---
+
+We recently announced the [second major release of Gatsby](/blog/2018-09-17-gatsby-v2/) 🚀. One change we want to highlight is the switch to using [@reach/router](https://reach.tech/router) to improve the accessibility of routing in Gatsby sites. What is @reach/router, and what are the benefits of undertaking the switch?
+
+## What is @reach/router?
+
+@reach/router is a routing library for React written and maintained by one of the original creators of [react-router](https://github.com/ReactTraining/react-router), [Ryan Florence](https://twitter.com/ryanflorence). After [investigating the tradeoffs](https://github.com/gatsbyjs/gatsby/issues/5656), we made the leap to depending on @reach/router:
+
+- [Accessibility](https://reach.tech/router/accessibility) is a first-class concern
+- Smaller bundle size (~70% decrease, 18.4kb to 6kb gzipped)
+- Simplified API
+
+## How does it support accessibility?
+
+When you visit a Gatsby site, a static, server-rendered HTML page is loaded first, and then the JavaScript to hydrate the site into a web app is loaded. From there, internal routing is handled with the [Gatsby Link component](/docs/gatsby-link/), which wraps [@reach/router’s Link component](https://reach.tech/router/api/Link).
+
+Web apps rerender in the client -- without making a request to the server to fetch new HTML -- resulting in a faster, more seamless user experience. The performance benefits associated with not having to fully load a new page, however, can come at the expense of a broken experience for users who rely on assistive technology like screen readers.
+
+When navigating between traditional server-rendered pages, the page is fully reloaded and in response, screen readers can announce the new content. When using most client-side routing solutions (out of the box), without the page reload, screen readers don’t know new content has been loaded to focus or announce.
```

Comment:
I think it would be cool to make this personal, like "Let's say you're visiting a single page app using screen reading technology. Here's how the experience would feel without reach-router...

And here's how it would feel with reach-router...

Or if that's straying too far from the technical details that the audience would be interested in, I think the beginning of this paragraph could be clearer. "When a person is navigating" would make the subject clearer.

Reply:
So this is why I included the video example. I could pretty much narrate that, but I didn't think that would add much.

Reply:
Ok, then I'd just recommend the second idea, "When a person is navigating" to make the subject clear, because as is, "the page" could actually be the subject. Which would be cool, if pages could navigate other pages...maybe someday? ;b